packages/app/src/app/pages/common/Modals/StorageManagementModal/FilesList/index.js

```diff
@@ -71,10 +70,10 @@ class FilesList extends Component {
                 `}
               />
               <HeaderTitle>File</HeaderTitle>
-              <HeaderTitle>Created</HeaderTitle>
```

Comment:
What's happening here?

Reply:
The date was not coming back from the server :(

it was always undefined